pilot/pkg/model/config.go

```diff
@@ -477,6 +477,26 @@ var (
 		Validate:    ValidateEndUserAuthenticationPolicySpecBinding,
 	}
 
+	// ServiceRole describes an RBAC service role.
+	ServiceRole = ProtoSchema{
+		Type:        "service-role",
+		Plural:      "service-roles",
+		Group:       "config",
+		Version:     istioAPIVersion,
```

Comment:
May be you should use separate api group (e.g 'authorization'), with your actual api version + @xiaolanz

Reply:
Hey, this has to be in "rbac" and v1alpha1.

Reply:
It's not a new CRD, it's currently defined [here](https://github.com/istio/istio/blob/4ac60af504902674f1452848d4110ffbaaf571e0/install/kubernetes/templates/istio-mixer.yaml.tmpl#L587) (group='config' and version='v1alpha2').